site/en/posts/iwd-2022-julia-enthoven.md

```diff
@@ -81,6 +85,10 @@ There are so many things! Well, one is to don't underestimate the power of belie
 
 The second thing. I would say, in everything you do in life, you just have to enjoy the journey. That's also been so important to me with Kapwing too. Have fun along the way. Don't stress too much. Try to bring levity into your work and life. Having a spirit of abundance and gratitude for everything you do is so important.
 
+> If you ever catch yourself thinking that you’re just not good at that thing, try and recognize and reverse that thinking. Because it will drain your energy. It’s especially important for women in tech, because there are so many opportunities for you to feel like you’re just not cut out for it or not capable. And it's just not true.
+
+Julia Enthoven, co-founder and CEO of Kapwing {.cite}
```

Comment:
```suggestion
> If you ever catch yourself thinking that you’re just not good at that thing, try and recognize and reverse that thinking. Because it will drain your energy. It’s especially important for women in tech, because there are so many opportunities for you to feel like you’re just not cut out for it or not capable. And it's just not true.
> {cite="Julia Enthoven, co-founder and CEO of Kapwing"}
```

Reply:
Thanks for changing to the new citations. I like the way the new one looks! But I think as per Natalia's comment, it's a bit redundant, so I decided to just remove them altogether.